packages/storybook-utils/src/actions.ts

```diff
@@ -23,6 +23,40 @@ export const enhanceEventArgTypes: ArgTypesEnhancer = ({ argTypes }) => {
   return argTypes;
 };
 
+/**
+ * Allows logging and documentation for the passed event listener names in Storybook.
+ * Will be documented in a extra "Relevant HTML events" section in the Storybook documentation.
+ *
+ * @example
+ * ```typescript
+ * const meta: Meta<typeof OnyxButton> = {
+ *   title: "Buttons/Button",
+ *   component: OnyxButton,
+ *   argTypes: {
+ *     somethingElse: { ...someOtherArgType },
+ *     ...withNativeEventLogging(["onClick"]),
+ *  },
+ *};
+ * ```
+ *
+ * @param relevantEvents a list of event names that should be logged
+ * @returns Storybook ArgTypes object
+ */
+export const withNativeEventLogging = (relevantEvents: (keyof Events)[]) =>
+  relevantEvents.reduce((argTypes, eventName) => {
+    const action = eventName.replace(/^on/, "").toLowerCase();
+    argTypes[eventName] = {
+      name: eventName,
```

Comment:
```suggestion
      name: action,
```

Would prefer to show the event with the on prefix